webapp/packages/core-blocks/src/ObjectPropertyInfo/evaluate.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2025 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import subscript from 'subscript';
+import { err, token } from 'subscript/parse';
+
+import 'subscript/feature/bool.js';
+
+const UNDEFINED_MAPPER_VALUE = new Array(2).fill(undefined, 1);
+token('undefined', 20, a => (a ? err() : UNDEFINED_MAPPER_VALUE));
```

Comment:
why not use `token('undefined', 20, a => a ? err() : [, undefined])` as in original example?
https://github.com/dy/subscript?tab=readme-ov-file#extending

Reply:
Eslint error here [, undefined] because of the comma